go/service/main.go

```diff
@@ -561,7 +561,7 @@ func (d *Service) writeServiceInfo() error {
 }
 
 func (d *Service) chatEphemeralPurgeChecks() {
-	ticker := libkb.NewBgTicker(5 * time.Minute)
```

Comment:
Will this tight loop stop the CPU from going quiescent on e.g. a small laptop? Would it be possible to calculate the minimum next purge time, and sleep for that much time?

Reply:
@mmaxim what's your intuition about loops vs. the battery? Do we already have other loops like this?

Reply:
Yeah I'm not super excited about this, I thought it was supposed to be 30 seconds?

Reply:
oh i misunderstood -- i suggested 30s but thought you then said 1s, i'll change it. we can try to do the minimum next purge time but we wanted to avoid added complexity. let's discuss tomorrow..